x/evm/types/params.go

```diff
@@ -62,17 +64,19 @@ func NewParams(
 	config ChainConfig,
 	extraEIPs []int64,
 	activePrecompiles,
-	evmChannels []string,
+	evmChannels,
+	activeDynamicPrecompiles []string,
```

Comment:
Use spread operator

Reply:
why? here we just need the pointer to instantiate the struct